include/psa/crypto_extra.h

```diff
@@ -1224,6 +1224,20 @@ static void psa_pake_cs_set_primitive(
                            psa_pake_primitive_t primitive
                            );
 
+/** Retrieve the PAKE family from a PAKE cipher suite.
+ *
+ * This function may be declared as `static` (i.e. without external
+ * linkage). This function may be provided as a function-like macro,
+ * but in this case it must evaluate its argument exactly once.
```

Comment:
This is something that is useful in the PSA Crypto API specification, but in Mbed TLS we might not want it. Did you find this description in the PSA Crypto API specification?

Reply:
@yanesca I copied this from other psa_pake_cs_set_XXX documentation

Reply:
Oh, I see, then it is better to be consistent.

Reply:
If we really wanted to be consistent, we'd add it to other functions in `psa/crypto.h` and `psa/crypto_extra.h`. But of course it would be out of scope here.

In Mbed TLS, I don't think this paragraph is particularly useful. If a function has external linkage, it's no big deal to keep it with external linkage even if it becomes a trivial wrapper around another function in the future. I don't remember encountering a case where we wanted to make a function static but held back due to a concern about backward compatibility.

Reply:
In that case, let us remove the paragraph about linkage from everywhere. (As it is not very useful and we don't have these anywhere else just in PAKE).

Reply:
Done, I removed those

Reply:
Thanks!